src/components/products/Products.jsx

```diff
@@ -0,0 +1,16 @@
+import "./products.css";
+import { ProductCard} from "./ProductCard";
+
+export default function Products(props) {
```

Comment:
Try to be consistent. Either use traditional Functions or arrow function

Reply:
sure, I will be mindful next. Thanks